src/cursor/cur_std.c

```diff
@@ -756,19 +756,36 @@ __wt_cursor_cache_release(WT_SESSION_IMPL *session, WT_CURSOR *cursor, bool *rel
     return (ret);
 }
 
+/*
+ * __wt_cursor_get_hash --
+ *     Set hash value from the given uri.
```

Comment:
The description needs to change - "Get hash..." rather than "Set hash..."

Reply:
Fixed